README.md

```
@@ -1,37 +1,74 @@
+  # Solve the conflicts.
+  # Make sure packages/excalidraw/package.json has the correct package version number.
+  # example: 0.18.0-alkemio-2
+
+  # Then push
+  git push --set-upstream origin <new-branch-name>
+  # example:
+  git push --set-upstream origin 0.18.0-alkemio-2
+```
+Create a Pull Request to develop in excalidraw-fork repository
+
 ### Build and publish the new npm package:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)